operations/conman/ConMan.md

```diff
@@ -2,12 +2,13 @@
 
 ConMan is a tool used for connecting to remote consoles and collecting console logs. These node logs can then be used for various administrative purposes, such as troubleshooting node boot issues.
 
-ConMan runs on the system as a containerized service. It runs in a set of Docker containers within Kubernetes pods named `cray-console-operator` and `cray-console-node`. ConMan can be configured to encrypt usernames and passwords. Node console logs are stored locally within the `cray-console-operator pod` in the `/var/log/conman/` directory, as well as being collected by the System Monitoring Framework \(SMF\).
+ConMan runs on the system as a containerized service. It runs in a set of Docker containers within Kubernetes pods named `cray-console-operator` and `cray-console-node`.
+ConMan can be configured to encrypt usernames and passwords.
+Node console logs are stored locally within the `cray-console-operator pod` in the `/var/log/conman/` directory, as well as being collected by the System Monitoring Framework \(SMF\).
```

Comment:
Change 'cray-console-operator pod' to 'cray-console-node-0 pod'.
Both are true (they point to the same mount), but in csm-1.3 the access will be removed from cray-console-operator so may as well start changing docs now.